drydock/plugin.py

```diff
@@ -28,12 +28,14 @@
         "NEWRELIC_LICENSE_KEY": "",
         "CUSTOM_CERTS": {},
         "DEBUG": False,
+        "LETSENCRYPT_EMAIL": None,
```

Comment:
```suggestion
        "LETSENCRYPT_EMAIL": "{{ CONTACT_EMAIL }}",
```
Wouldn't be better to set the the default here instead of doing it in the template?